src/components/Adyen/Adyen.js

```diff
@@ -74,28 +82,37 @@ const Adyen = ({
   const getBankCopy = () => {
     const isFree = totalPrice === 0;
     const isSubscription = offerId?.charAt(0) === 'S';
+    // TODO before the release: standarize transaltions keys => done, to be checked
+    // TODO: test if purchase is blocked if checkbox is not checked => done, works
+    // TODO: add copy and checkbox to PayPal => done, works, used getStandardCopy
 
     if (isMyAccount || (isFree && isSubscription)) {
+      // TODO: add link to T&C
       return t(
         'offer-bank-consent-copy.free-subscription',
-        'You accept the terms and conditions of this agreement and that your account will be charged €0.10 for authentication purposes. This amount will be refunded once the transaction is completed. And your account will be debited on a recurring basis for the full subscription amount.'
+        `By ticking this, you agree to the Terms and Conditions of our service. Your account will be charged €0.10 for authentication purposes. This amount will be refunded once the transaction is completed. Your account will be charged on a recurring basis for the full subscription amount. Your subscription will continue until you cancel.`
       );
     }
 
     if (isSubscription) {
+      // TODO: add link to T&C
       return t(
         'offer-bank-consent-copy.paid-subscription',
-        'You accept the terms and conditions of this agreement. Your account will be debited on a recurring basis for the full subscription amount.'
+        'By ticking this, you agree to the Terms and Conditions of our service. Your account will be charged on a recurring basis for the full subscription amount. Your subscription will continue until you cancel.'
       );
     }
 
     return t(
+      // TODO: add link to T&C
       'offer-bank-consent-copy.paid-not-subscription',
-      'You accept the terms and conditions of this agreement.'
+      'By ticking this, you agree to the Terms and Conditions of our service.'
     );
   };
 
-  const addAdditionalCopyForBankPaymentMethods = methodName => {
+  const addAdditionalCopyForBankPaymentMethods = (
```

Comment:
as this is no longer just for bank payment method we should change the name of it, eg. 
`addLegalCheckboxForPaymentMethod`